RELEASE.rst

```diff
@@ -0,0 +1,16 @@
+RELEASE_TYPE: patch
+
+This release fixes some minor bugs in argument validation:
+
+    * `hypothesis.extra.numpy` dtype strategies would raise an internal error
```

Comment:
I'm not sure that this link will work as you expect - check, and if not either make it a literal or use a link to the `hypothesis-numpy` link target with custom text.